### ISSUE TITLE
Update devguide regarding the new Python Triagers team

### ISSUE THREAD

**Issue by Mariatta** [python/core-workflow#339]
Document the role and expectations

**Comment by Mariatta** [comment]
Related discussion:. https://discuss.python.org/t/proposal-create-bug-triage-team-on-github/992

Also document that exsiting triagers in bpo are welcome to request themselves be added to the GitHub triagers team.

**Comment by aeros167** [comment]
I wouldn't mind helping with this. Do you think that it should be an expansion upon the already existing section https://devguide.python.org/triaging/ ?

**Comment by Mariatta** [comment]
Thanks. Sounds good to me to expand that section, or adding a new section of just "triaging pull request on GitHub".

**Comment by aeros167** [comment]
Alright, I'll create a new section towards the end of the triaging page for now (with a link at the top for easy reference) and create a PR over on `python/devguide` (linking to this issue) after typing up a draft, based on the proposal and discussion over at https://discuss.python.org/t/proposal-create-bug-triage-team-on-github/992.

**Comment by aeros167** [comment]
Instead of explaining it in the context of triaging on GitHub, my PR focused around explaining the role of the Bug Triager (with the expectations on GitHub for the role in bullet points). The details of the each responsibility could definitely be expanded upon further if needed, or there could be a separate section which describes the github review process. This initial draft just provided an overview based on the information in the proposal rather than an in-depth GitHub triaging guide.

**Comment by aeros167** [comment]
@Mariatta As far as naming for the role itself goes, is the preference towards `Python Triager` , `Bug Triager`, or something else? I mostly just used `Bug Triager` since the name of the team on the proposal on the discussion boards was the `Bug Triage` team. I can edit the PR accordingly.

**Comment by Mariatta** [comment]
I'm not sure what the name should be :) I asked in #334

**Comment by Mariatta** [comment]
another issue related to this is https://github.com/python/devguide/issues/296
It would be great to explain the purpose of each GitHub labels. The triage section explains each components of the issue tracker clearly, but nothing has been written about our GitHub labels yet.

**Comment by aeros167** [comment]
> It would be great to explain the purpose of each GitHub labels.

Agreed, this would be a good idea for the next component to start working on, once the role summary section is finished.

**Comment by Mariatta** [comment]
Closed via https://github.com/python/core-workflow/issues/339
